lib/thread.c

```diff
@@ -45,6 +45,15 @@ DEFINE_MTYPE_STATIC(LIB, THREAD_STATS, "Thread stats")
 
 DECLARE_LIST(thread_list, struct thread, threaditem)
 
+/*
+ * By default collecting extra thread statistics like max-wakeup-interval
+ * and last-cpu-hog are disabled.
+ *
+ * One may use thread_set_stats_collection() to enable it at the time
+ * of initializing any specific daemon.
+ */
+static int thread_collect_stats = false;
```

Comment:
no, please - if it's "false" or "true", it's a boolean?

Reply:
@mjstapp false is actually #defined to 0. So I did not get the concern here.

Reply:
static bool thread_collect_stats

Reply:
Ok. If you prefer so. :)

Reply:
Don't initialize statics to false per kernel style